berbix/__init__.py

```diff
@@ -7,7 +7,7 @@
 from requests.exceptions import HTTPError
 
 
-SDK_VERSION = '0.0.13'
+SDK_VERSION = '0.0.14'
```

Comment:
Minor note: we generally do the version bump when we merge and release. Not an issue here, though, given that we do not have other outstanding PRs.